e2e/cypress/integration/preview.spec.mjs

```diff
@@ -0,0 +1,36 @@
+describe('previews API', () => {
```

Comment:
our intentions is to test UI flow with `e2e`, api testing should be covered by `integration` tests from previous task